b9/include/b9/instructions.hpp

```diff
@@ -70,6 +70,12 @@ enum class OpCode : RawOpCode {
   // Jump if two strings are not equal
   STR_JMP_NEQ = 0x17,
 
+  //Push from function parameter
+  PUSH_FROM_ARG = 0x18,
+
+  //Pop into a parameter
+  POP_INTO_ARG = 0x19,
+
```

Comment:
I know this is a lot of tedious work, but can we have the push/pop from arg come right after (or before) the var instructions?